Docs release 1.9.0 csi 3911 node get volume stats (#460)
IBM/ibm-block-csi-driver: docs/content/release_notes/whats_new.md

Stretched volumes and stretched snapshots (FlashCopy) are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).

## New metrics support

IBM® block storage CSI driver 1.9.0 introduces new kubelet volume metric support. Use 

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Author: Is this more correct?

"Use the `NodeGetVolumeStats` Remote Procedure Call (RPC) call to retrieve the volume metrics with a metrics scraper."

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Reviewer: no, the verb "use" addresses the user, but a user cannot directly issue remote procedure calls.
possible metrics info flows:
`csi node` --gRPC--> `kubelet` --HTTP--> `user`
`csi node` --gRPC--> `kubelet` --HTTP--> `prometheus` --?--> `user`
invalid flow:
`csi node` --gRPC--> `user`
@matancarmeli7 is the feature owner, he can elaborate

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Author: @matancarmeli7  / @oriyarde  Please help provide an appropriate sentence that I can work with.

Maybe more like - 
"The `NodeGetVolumeStats` Remote Procedure Call (RPC) call enables retrieval of the volume metrics with a metrics scraper."?

or maybe we don't need this sentence all together? Do we need to mention NodeGetVolumeStats?

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Reviewer: that is correct, but as your questions suggest, I agree we don't need to mention `NodeGetVolumeStats`, that is an implementation detail

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Author: is the link that we provide enough information for implementation?

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Reviewer: yes, though I think the link I wrote [here](https://github.com/IBM/ibm-block-csi-driver/pull/460#discussion_r817495108) could be useful for how to use this feature

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Reviewer 2: IMO we don't need to mention `NodeGetVolumeStats`, we need to say something like this: "We are supporting exposing various metrics through `kubelet` on mounted volumes that have been created by IBM block CSI" (the english can be better, but this is the direction I think is good) and add the relevant metrics, link that Ori sent [here](https://github.com/IBM/ibm-block-csi-driver/pull/460#discussion_r817495108) and the spec link that we already provided. 
What do you think?

> `NodeGetVolumeStats` to retrieve volume metrics. The following metrics are currently supported:

Author: updated - please approve

- kubelet_volume_stats_available_bytes
- kubelet_volume_stats_capacity_bytes
- kubelet_volume_stats_inodes
- kubelet_volume_stats_inodes_free
- kubelet_volume_stats_inodes_used
- kubelet_volume_stats_used_bytes

> For more information on using kubelet volume metrics, see the [Container Storage Interface (CSI) spec documentation for `NodeGetVolumeStats`](https://github.com/container-storage-interface/spec/blob/v1.5.0/spec.md#nodegetvolumestats).

Reviewer: ```suggestion
For more information about each metric, see `VolumeUsage` in the [Container Storage Interface (CSI) spec documentation for `NodeGetVolumeStats`](https://github.com/container-storage-interface/spec/blob/v1.5.0/spec.md#nodegetvolumestats).
```

Author: updated
